### ISSUE TITLE
Issue with VASP_VDW_KERNEL_DIR Setup Not Locating vdw_kernel.bindat File

### ISSUE THREAD

**Issue by YuanbinLiu** [materialsproject/atomate2#959]
Hi,

I have been working on setting up Atomate2 for my VASP calculations, and I encountered an issue related to the VASP_VDW_KERNEL_DIR variable. I followed the instructions provided in the Atomate2 documentation (available [here](https://materialsproject.github.io/atomate2/reference/atomate2.settings.Atomate2Settings.html#atomate2.settings.Atomate2Settings.VASP_VDW_KERNEL_DIR)) to configure the VASP_VDW_KERNEL_DIR variable. The goal was to ensure that the vdw_kernel.bindat file could be correctly located during my VASP calculations.

Despite following the documentation, it appears that the vdw_kernel.bindat file is still not being found during the calculations. I have double-checked the path and confirmed that the file exists at the specified location.

Could you please provide some guidance on what might be going wrong? Specifically:

1. Are there any common pitfalls or additional steps that I might have overlooked in setting up this variable?
2. Is there a way to verify within Atomate2 that the VASP_VDW_KERNEL_DIR variable is correctly pointing to the intended directory?

**Comment by utf** [comment]
Hi @YuanbinLiu, can I just check if you're using a relative or absolute path, and whether you're using `~/` or putting the full path.

**Comment by JaGeo** [comment]
@utf I had just looked for the constant in the code base and only found it showing up twice: https://github.com/search?q=repo%3Amaterialsproject%2Fatomate2+VDW_KERNEL_DIR&type=code

I am currently not sure how and where it should be used. I assume I am missing something here.

**Comment by YuanbinLiu** [comment]
> Hi @YuanbinLiu, can I just check if you're using a relative or absolute path, and whether you're using `~/` or putting the full path.

Hi @utf, I am using the absolute/full path.

**Comment by JaGeo** [comment]
Sorry for closing. This was a mistake from my side.

**Comment by utf** [comment]
I agree @JaGeo. It looks like this option doesn't currently do anything. I think it would make sense to actually have this option in pymatgen and then automatically copy the kernel when writing the input set.

@YuanbinLiu Do you have access to VASP v6.4.3? If so, the kernel dir is not necessary as described on this page: https://www.vasp.at/wiki/index.php/Nonlocal_vdW-DF_functionals

**Comment by YuanbinLiu** [comment]
@utf I have access. Implementing this feature in the future would enhance the program's flexibility.

**Comment by utf** [comment]
@YuanbinLiu I agree - I'll look into this. It may take some time though, so it will be easiest and quickest if you use the updated version.

**Comment by YuanbinLiu** [comment]
> @YuanbinLiu I agree - I'll look into this. It may take some time though, so it will be easiest and quickest if you use the updated version.

Thank you very much!

**Comment by Andrew-S-Rosen** [comment]
Can confirm `VASP_VDW_KERNEL_DIR` does nothing. It should be removed until it does something so as to not confused users.
